os/test/src/SubprocessTests.scala

```diff
@@ -82,7 +82,20 @@ object SubprocessTests extends TestSuite {
             env = Map("ENV_ARG" -> "123")
           )
 
-        assert(res.out.text().trim() == "Hello123")
+        // Enhanced debugging: show exit code and raw output on failure
```

Comment:
I think these changes will help debugging a lot.  Thank you.

I'll check the logs of the next os-lib CI run and see what I can
glean.